.env.test

```diff
@@ -7,3 +7,12 @@ SECRET_KEY = "491a167e67abe9aa3d40e1e4"
 token = "4c82ed8e16b482daf8c417cc0f1187bf943dc7d3dcbd212d08b39b51a5c58847"
 api_key = "38562d78c91237d032fc48f4301cab52"
 TRELLO_BOARD_ID = "34"
+FLASK_APP=todo_app/app
+FLASK_ENV=development
+FLASK_SKIP_DOTENV=True
+
+SECRET_KEY = "491a167e67abe9aa3d40e1e4"
+
+token = "4c82ed8e16b482daf8c417cc0f1187bf943dc7d3dcbd212d08b39b51a5c58847"
+api_key = "38562d78c91237d032fc48f4301cab52"
+TRELLO_BOARD_ID = "34"
```

Comment:
You seem to have accidentally duplicated everything in this file?


Also a heads up for the next module - Docker is stricter than python-dotenv about the format of this file. Each env var should be given in the form `ENV_VAR_NAME=some-value` - no quotes and no spaces around the equals sign.